compiler/AST/AstToText.cpp

```diff
@@ -1309,6 +1368,109 @@ void AstToText::appendExpr(NamedExpr* expr, bool printingType)
   appendExpr(expr->actual, printingType);
 }
 
+void AstToText::appendExpr(IfExpr* expr, bool printingType)
+{
+  mText += "if ";
+  if (Expr* sel = toExpr(expr->getCondition()))
+  {
+    appendExpr(sel, printingType);
+
+    mText += " then ";
+    if (BlockStmt* thenBlockStmt = toBlockStmt(expr->getThenStmt()))
+    {
+      if (thenBlockStmt->body.length == 1)
+      {
+        Expr* exp = thenBlockStmt->body.get(1);
+        appendExpr(exp, printingType);
+
+        mText += " else ";
```

Comment:
I don't think it is valid to have only "then" without "else". Because when I tried to build `chpldoc` for statements which has `if something then something;` then it returns me `syntax error: near ';'
`. Anyway, I'll move it to the inside the below if block.

Reply:
Ah, okay.  Thanks!